index.html

```diff
@@ -1,44 +1,226 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio</title>
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="style.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
 </head>
+
 <body>
-    <div id="container">
-        <div id="template1">
-            <header>
-                <a href="#" class="logo">My Logo</a>
-                <nav class="Union">
-                    <img src="images/header/Union.png" alt="bar">
-                </nav>
-            </header>
-            <div class="bg_image"></div>
-                <section id="landing_page">
-                    <div class="intro">
-                        <h1>I’m Praveen Glad to see you!</h1>
-                    </div>
-                    <div class="description">
-                        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                    </div>
-                    <div class="connect">
-                        <h3>LET'S CONNECT</h3>
-                    </div>
-                    <ul>
-                        <li><a href=""><img class="twitter" src="images/header//twiter.png" alt="twiter_logo"></a></li>
-                        <li><a href=""><img class="linkedIn" src="images/header//linkedin.png" alt="linkedIn_logo"></a></li>
-                        <li><a href=""><img class="m" src="images/header//m.png" alt="m_logo"></a></li>
-                        <li><a href=""><img class="github" src="images/header//github.png" alt="github_logo"></a></li>
-                        <li><a href=""><img class="ok" src="images/header//ok.png" alt="ok_logo"></a></li>
+    <header class="tool_bar">
+        <nav class="the_nav">
+            <a href="#" class="my_logo"><img src="images/header/desktop/myLogo.png" alt="Wali's-logo"></a>
```

Comment:
As the last [code reviewer suggested](https://github.com/Praveen1995-10-14/myportfolio/pull/10#discussion_r990641505) kindly fix the src property of this tag because the image is not being rendered. :eyes: :eyes:  
![image](https://user-images.githubusercontent.com/98363075/194715412-44309c85-dd01-4599-8cda-883531751fa7.png)

Reply:
could you please check again I made the changes that you told me